applications/callflow/src/module/cf_directory.erl

```diff
@@ -138,6 +139,11 @@ handle(Data, Call) ->
             cf_exe:continue(Call)
     end.
 
+-spec directory_start_sorted_users(kapps_call:call(), directory(), directory_users()) -> 'ok'.
+directory_start_sorted_users(Call, State, CurrUsers) ->
+    Users = sort_users(CurrUsers, State#directory.sort_by),
+    directory_start(Call, State, Users).
+
 -spec directory_start(kapps_call:call(), directory(), directory_users()) -> 'ok'.
 directory_start(Call, State, CurrUsers) ->
     directory_start(Call, State, CurrUsers, 3).
```

Comment:
What do you think about putting the sort call here (on `CurrUsers`); smaller changeset mostly :)

Reply:
Also, as a general preference of mine:

I like to pattern-match records in the function header instead of using the `Var#record.field` syntax. So `directory_start_sorted_users(Call, #directory{sort_by=SortBy}=State, CurrUsers) ->` then use `SortBy` in the function clause.